### PR TITLE
Add Discord Button to the website 

### PR DESCRIPTION
**Pull request by ryuzaki221**
Issue: 
#643 

Created `_discord.less` file and added the button to website via `assets/js/button-examples.js`

### CODE REVIEW

File: assets/js/buttons-examples.js [hunk 1]
```diff
@@ -363,6 +363,10 @@ $(document).ready(function () {
           classes: "weibo",
           text: "Login with Weibo",
         },
+        {
```
Comment (shahednasser) [inline assets/js/buttons-examples.js:366]: Place it in the correct alphabetical order under "Social"

**Review by shahednasser — CHANGES_REQUESTED** [review]
Great work! Just make the change below.

**Comment by ryuzaki221** [comment]
Sorry about that error. I placed it at the right place.

**Comment by shahednasser** [comment]
Great thank you!

@all-contributors please add @ryuzaki221 for code and idea

**Pull request merged**